packages/orchestrator/lib/clients/types.ts

```diff
@@ -70,6 +71,7 @@ interface TaskCommonFields {
     groupKey: string;
     state: TaskState;
     attempt: number;
+    attemptMax: number;
```

Comment:
exposing the attemptMax so we can show it in the logs and decide to fail the log operation if last attempt